### PR TITLE
Widescreen lobby art

### PR DESCRIPTION
**Pull request by S34NW**
## What Does This PR Do
Widens the existing lobby art to fit widescreen using the various tools available in Photoshop. The "red light" version had to be re-made from the base version so the colour gradient is slightly off. No way to match it perfectly without source files.

Adds the ability to dynamically center lobby art based on the icon size, so any future lobby art can just be dropped right in even if the size is, for some reason, not perfect.

(Does not apply to var-edited lobby art that is not the full screen size)

## Why It's Good For The Game
Walls are not fun

## Images of changes
OLD:
image

NEW:
default

## Changelog
:cl:
tweak: Widened the existing lobby art for compatibility with widescreen
/:cl:

### CODE REVIEW

**Comment by AffectedArc07** [comment]
What does this look like on all 3 screen sizes?

**Comment by S34NW** [comment]
> What does this look like on all 3 screen sizes?

image

looks like you went the wrong route with making the existing start screen be widescreen-compat... This is gonna need some tweaks before it is ready.

File: code/game/turfs/simulated/walls_indestructible.dm [hunk 1]
```diff
@@ -110,6 +110,9 @@
 	icon_state = ""
 	layer = FLY_LAYER
 	flags = NO_SCREENTIPS
+	// Pixel shifts below are needed to centrally position the lobby art within the start area for all screen sizes
```
Comment (S34NW) [inline code/game/turfs/simulated/walls_indestructible.dm:113]:
```suggestion
	// Pixel shifts below are needed to centrally position the black placeholder icon within the start area at compile-time. This is overridden when a "real" lobby art image is chosen by SStitlescreen
```

**Review by SteelSlayer — APPROVED** [review]
Very nice

**Review by hal9000PR — APPROVED** [review]

**Pull request merged**